test/test-utils.js

```diff
@@ -27,8 +27,8 @@ function generateUniqueString(prefix) {
     return `${prefix}-${Date.now()}-${Math.round(Math.random() * 1000000)}`;
 }
 
-async function createAccount(near) {
-    const newAccountName = generateUniqueString('test');
+async function createAccount(near, str = 'test') {
```

Comment:
not sure what is the reason for the change? Does it matter for test what is account name prefix?